game-core/src/main/java/games/strategy/engine/data/properties/GameProperties.java

```diff
@@ -99,7 +99,7 @@ public String get(final String key, final String defaultValue) {
     if (value == null) {
       return defaultValue;
     }
-    return (String) value;
+    return String.valueOf(value);
```

Comment:
This adds flexibility if we want to load an integer property as a string, otherwise we get a cast exception